hw/xbox/xid-sdl.c

```diff
@@ -464,6 +478,63 @@ static void usb_xid_class_initfn(ObjectClass *klass, void *data)
     uc->handle_attach  = usb_desc_attach;
 }
 
+static HapticEmulationMode pick_haptic_emulation_mode(SDL_Haptic *haptic)
+{
+    if (SDL_HapticQuery(haptic) & SDL_HAPTIC_LEFTRIGHT) {
+        return EMU_HAPTIC_LEFT_RIGHT;
+    }
+
```

Comment:
nit: For debug / user-support purposes, we could print the query result as hex value, so we can quickly decide on a plan of action if users report broken haptic for their device.